core/src/main/scala/org/apache/spark/api/python/PythonUtils.scala

```diff
@@ -27,12 +27,15 @@ import org.apache.spark.SparkContext
 import org.apache.spark.api.java.{JavaRDD, JavaSparkContext}
 
 private[spark] object PythonUtils {
+  val PY4J_ZIP_NAME = "py4j-0.10.7-src.zip"
+
   /** Get the PYTHONPATH for PySpark, either from SPARK_HOME, if it is set, or from our JAR */
   def sparkPythonPath: String = {
     val pythonPath = new ArrayBuffer[String]
     for (sparkHome <- sys.env.get("SPARK_HOME")) {
       pythonPath += Seq(sparkHome, "python", "lib", "pyspark.zip").mkString(File.separator)
-      pythonPath += Seq(sparkHome, "python", "lib", "py4j-0.10.7-src.zip").mkString(File.separator)
+      pythonPath +=
+        Seq(sparkHome, "python", "lib", PY4J_ZIP_NAME).mkString(File.separator)
```

Comment:
This line looks shorter than the existing line? Do we need to split?

Reply:
Yup, just in case we backport other changes. I think it's better to keep the diff small comparing to the master.

Reply:
I see. Got it.